schema/deploy/roles/materialized-view-refresher/grants.sql

```diff
@@ -0,0 +1,43 @@
+-- Deploy seattleflu/schema:roles/materialized-view-refresher/grants to pg
+-- requires: roles/materialized-view-refresher/create
+
+begin;
+
+revoke all on database :"DBNAME" from "materialized-view-refresher";
+revoke all on schema receiving, warehouse, shipping from "materialized-view-refresher";
+revoke all on all tables in schema receiving, warehouse, shipping from "materialized-view-refresher";
+
+grant connect on database :"DBNAME" to "materialized-view-refresher";
+
+-- Regular users cannot refresh materialized views because only the view
+-- owner has permission to refresh them.
+-- Create a function that runs in the security context of its owner so that
+-- other users can refresh the view.
+create or replace function public.refresh_materialized_view(view_schema text, view_name text)
```

Comment:
I was a bit surprised to see the function definition in this grants file. I think I know why you did so; not sure I have a better solution. It might be a little less weird to move the function create and execute grants to _functions/refresh-materialized-view_?

Reply:
I don't really have a good reason for this. I was following the the grants set up of the [idle-session-disconnector](https://github.com/seattleflu/id3c-customizations/blob/master/schema/deploy/roles/idle-session-disconnector/grants.sql).

Reply:
Ah, ok. I think I probably missed that the function was inside the grants change in that PR. :-)

One reason for co-locating the grants to the function with the function definition itself is that reworking either doesn't create awkward inter-change dependencies in sqitch. I think there's room for improvement in some of the sqitch patterns we adopted early on (generally at my suggestion or example, so mea culpa), but I haven't had time to write up how I think we could do things differently.

That's all to say, this is fine for now. :-)